components/backup/src/endpoint.rs

```diff
@@ -373,7 +375,14 @@ impl<E: Engine, R: RegionInfoProvider> Endpoint<E, R> {
                     warn!("backup task has canceled"; "range" => ?brange);
                     return Ok(());
                 }
-                let name = backup_file_name(store_id, &brange.region);
+                let table_id = brange.start_key.clone().and_then(|k| {
+                    if let Ok(t_id) = decode_table_id(&k.into_raw().unwrap()) {
```

Comment:
It's better to call `decode_table_id(&k.into_raw().unwrap()).ok()`.

https://doc.rust-lang.org/1.38.0/core/result/enum.Result.html#method.ok

Reply:
seems good, let me try it

Reply:
[Done](https://github.com/tikv/tikv/pull/5660/files#diff-53831868d19a8c3495a300c681e252a8R378-R382)